src/frontend/Layout/MainLayout.razor

```diff
@@ -1,8 +1,13 @@
 ﻿@inherits LayoutComponentBase
 
-<MudLayout>
-    <MudMainContent>
-        @Body
+
+<MudLayout Class="pt-3">
+    <Header/>
+    <MudMainContent Class="d-flex flex-row" Width="100vw">
```

Comment:
With, as it is not marked btw, is not a supported attribute on the MudMainContent Component

Reply:
dun